### DISCUSSION TITLE
Argument .column-page does not work with fig- prefix

### DISCUSSION THREAD

**Discussion by ManuelSpinola** [quarto-dev/quarto-cli#9834] [Q&A] [answered]
### Description

When I use the following  layout configuración the .column-page argument it does not work.  If I remove the fig- prefix the argument .column-page works.  I am using the last Quarto versión (1.4.554).

````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: knitr
---

:: {.column-page layout="[15,-1,15]" layout-valign="center" #fig-elephants}

![Surus](surus.png)

![Lin Wang](lin-wang.png)

Some elephants
:::
````

**Comment by mcanouil** [comment, marked as answer]
I believe this to be the same thing as the already reported issue:

- https://github.com/quarto-dev/quarto-cli/issues/8763

**Comment by ManuelSpinola** [comment]
Thank you very much.
